packages/auth/src/shared/utils.ts

```diff
@@ -7,3 +9,21 @@ export function generateNonce(): string {
 export function generateToken(): string {
   return crypto.randomBytes(10).toString('hex').toUpperCase()
 }
+
+export function generateRouteLogs(ctx: AppContext): {
+  requestId: string
+  route: typeof ctx.path
+  method: typeof ctx.method
+  params: typeof ctx.params
+  headers: typeof ctx.headers
+  requestBody: typeof ctx.request.body
+} {
+  return {
+    requestId: v4(),
```

Comment:
This would show a different uuid for each log, right? 
I was looking into a way to have a common request/traceId as a property on a child pino logger, but it is a bit difficult to do that with how we've set up dependency injection with adonis fold library. I'll take another attempt at addressing this in my upcoming changes.

Reply:
I figured it would be workable for now as the routes log once per request, but I admit it's wishful thinking